gensim/models/keyedvectors.py

```diff
@@ -189,7 +189,28 @@
 logger = logging.getLogger(__name__)
 
 
-KEY_TYPES = (str, int, np.integer)
+_KEY_TYPES = (str, int, np.integer)
+
+_EXTENDED_KEY_TYPES = (str, int, np.integer, np.ndarray)
+
+
+def _ensure_list(value):
+    #
+    # Ensure that the specified value is a list.  Sometimes users pass a single
+    # value when they should really pass a list containing value.
```

Comment:
Slightly off to say users "should" pass a list - when docs/examples have always encouraged other forms (& a single string key is probably the most common usage). Instead I'd say simply: "Ensure that the specified value is wrapped in a list, for those supported cases where we also accept a single key or vector."